packages/enricher/src/local-contexts-notices/definitions.ts

```diff
@@ -0,0 +1,35 @@
+import {basicEnrichmentBeingCreatedSchema, PubInfo} from '../definitions';
+import {z} from 'zod';
+
+// An enrichment can be about these Notices
+export enum LocalContextsNoticeEnrichmentType {
+  Authorization = 'https://localcontexts.org/notice/authorization/',
+  Belonging = 'https://localcontexts.org/notice/belonging/',
+  Caring = 'https://localcontexts.org/notice/caring/',
+  Gender_Aware = 'https://localcontexts.org/notice/gender-aware/',
+  Leave_Undisturbed = 'https://localcontexts.org/notice/leave-undisturbed/',
+  Safety = 'https://localcontexts.org/notice/safety/',
+  Viewing = 'https://localcontexts.org/notice/viewing/',
+  Withholding = 'https://localcontexts.org/notice/withholding/',
+}
+
+export const localContextsNoticeEnrichmentBeingCreatedSchema =
+  basicEnrichmentBeingCreatedSchema.merge(
+    z.object({
+      type: z.nativeEnum(LocalContextsNoticeEnrichmentType),
+    })
+  );
+
+export type LocalContextsNoticeEnrichmentBeingCreated = z.infer<
+  typeof localContextsNoticeEnrichmentBeingCreatedSchema
+>;
+
+export type LocalContextsNoticeEnrichment = {
+  id: string;
+  type: LocalContextsNoticeEnrichmentType;
+  description?: string;
+  citation?: string;
```

Comment:
In the design, there is no `citation`. I'm not sure if this is relevant for a Local Context Notice

Reply:
Yes, you're right. I've decided to add the property to be consistent with the other enrichment types. But if it's not needed in the frontend, we can ignore it.